pymc3/tests/test_smc.py

```diff
@@ -153,36 +156,36 @@ def abs_diff(eps, obs_data, sim_data):
 
     def test_one_gaussian(self):
         with self.SMABC_test:
-            trace = pm.sample_smc(draws=1000, kernel="ABC")
+            trace = pm.sample_smc(draws=1000)
```

Comment:
Oh, does `sample_smc` still return a `MultiTrace`? @OriolAbril

Reply:
Yes it still does. I am not sure how SMC should be mapped to inferencedata. It doesn't follow the traditional chains x samples. Maybe @aloctavia has some ideas.

Reply:
At least for the moment we should treat SMC output as we do with MCMC. So we should run more than one instance of the algorithm as a different "chain" (even if the samples are not markov chains). If we do this many diagnostics still work (at least qualitatively the same as with MCMC, @ricardoV94 I think I share with you some preliminary results about this.). Currently there is some information stored in "report" like the like the log_likelihood and the sequence of betas that we could store in  the `sample_stats` group,